ReadMe.md

```diff
@@ -51,10 +53,33 @@ setTimeoutAt(function (msg) {
 }, Date.now() + 5000, 'Hello world');
 ```
 
+**Timeout.clear() // clearTimeout(Timeout timeout)**
```

Comment:
I'd do a lowercase "t" on the `timeout.clear()`.  The uppercase "T" would suggest that it is a static method on the `Timeout` class.

Also, just one "/" between the two, please.